contracts/base/Dispatcher.sol

```diff
@@ -356,6 +351,19 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, Callbacks,
                 }
                 bytes calldata data = inputs.toBytes(1);
                 (success, output) = SEAPORT_V1_4.call{value: value}(data);
+            } else if (command == Commands.LOOKS_RARE_V2) {
+                // equivalent: abi.decode(inputs, (uint256, bytes))
+                uint256 value;
+                assembly {
+                    value := calldataload(inputs.offset)
```

Comment:
wondering whether this data/value thing gets repeated enough that a helper function might DRY things up?

Reply:
yeah nice idea. doing this in #276 woo